openwisp_radius/api/views.py

```diff
@@ -501,6 +501,7 @@ def post(self, request, *args, **kwargs):
                     'response_code': 'AUTH_TOKEN_VALIDATION_SUCCESSFUL',
                     'auth_token': token.key,
                     'radius_user_token': radius_token.key,
+                    'username': radius_token.user.username,
```

Comment:
The queryset above should be changed:

```python
UserToken.objects.select_related('user').get(key=request_token)
```

Can you please add an assertion that checks for the presence of this attribute in the response?
https://github.com/openwisp/openwisp-radius/blob/master/openwisp_radius/tests/test_api.py#L1401-L1407